wherehows-web/app/templates/components/search-bar-form.hbs

```diff
@@ -3,8 +3,9 @@
     <label for="search-input" class="sr-only">Search</label>
 
     {{aupac-typeahead
```

Comment:
I thought we were switching to power-select to standardize on one auto-complete across the app?

Reply:
I will, not in this PR :)